test-common/uk/gov/hmrc/economiccrimelevyregistration/EclTestData.scala

```diff
@@ -333,6 +335,20 @@ trait EclTestData {
       } yield ValidRegistrationWithRegistrationType(registration.copy(registrationType = Some(Initial)))
     }
 
+  implicit val arbValidRegistrationWithDifferentEntityTypes: Arbitrary[ValidRegistrationWithDifferentEntityTypes] =
```

Comment:
We don't need to make arbitraries here for all different types of scenarios, we could just use the copy function in the test itself?